core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -694,12 +705,18 @@ public NativeImageInvokerInfo build() {
                     nativeImageArgs.add("-H:-ParseOnce");
 
                     // AdditionalSecurityProviders
-                    if (nativeImageSecurityProviders != null && !nativeImageSecurityProviders.isEmpty()) {
+                    if (!nativeImageSecurityProviders.isEmpty()) {
                         String additionalSecurityProviders = nativeImageSecurityProviders.stream()
-                                .map(p -> p.getSecurityProvider())
+                                .map(NativeImageSecurityProviderBuildItem::getSecurityProvider)
                                 .collect(Collectors.joining(","));
                         nativeImageArgs.add("-H:AdditionalSecurityProviders=" + additionalSecurityProviders);
                     }
+
+                    // Add --exclude-config options
+                    for (ExcludeConfigBuildItem excludeConfig : excludeConfigs) {
+                        nativeImageArgs.add("--exclude-config");
+                        nativeImageArgs.add(excludeConfig.getValue());
```

Comment:
Hello @geoand, looking at the PR in graalvm that introduced this feature, I think the original comment/description (the top most one) in that PR is no longer accurate. From what I can see, that was the initial proposal which then got modified and from the looks of it, the way to configure this is now a "2 arg value" for that `--exclude-config`. Something like:

`--exclude-config <jar-file-name-or-regex> <resource-name-or-regex-within-that-jar>`

and the 2 args are mandatory. So I think this new `ExcludeConfigBuildItem` would need to be enhanced to allow that.

Reply:
Thanks @jaikiran! You are almost certainly right. 
I looked at the code a little and even tried to debug it, but for some reason I could not get the breakpoint in the code of the PR to hit so I was in the dark :)

I will update the PR based on your observation

Reply:
Fixed

Reply:
Thanks a lot!